src/components/@shared/FormFields/FilesInput/index.tsx

```diff
@@ -4,28 +4,33 @@ import { toast } from 'react-toastify'
 import FileInfo from './Info'
 import UrlInput from '../URLInput'
 import { InputProps } from '@shared/FormInput'
-import { getFileInfo } from '@utils/provider'
 import { useWeb3 } from '@context/Web3'
 import { getOceanConfig } from '@utils/ocean'
-import { useCancelToken } from '@hooks/useCancelToken'
 import { initialValues } from 'src/components/Publish/_constants'
+import { ProviderInstance } from '@oceanprotocol/lib'
+import axios, { Method } from 'axios'
 
 export default function FilesInput(props: InputProps): ReactElement {
   const [field, meta, helpers] = useField(props.name)
   const [isLoading, setIsLoading] = useState(false)
   const { chainId } = useWeb3()
-  const newCancelToken = useCancelToken()
 
   function loadFileInfo(url: string) {
     const config = getOceanConfig(chainId || 1)
 
     async function validateUrl() {
       try {
         setIsLoading(true)
-        const checkedFile = await getFileInfo(
+        const checkedFile = await ProviderInstance.checkFileUrl(
```

Comment:
we still have `getFileInfo()` in `src/@utils/provider.ts` so instead of adding another way of doing the same thing only here, would suggest rather modifying `getFileInfo()`. Cause we use `getFileInfo()` in other places so we should make sure when we get the file info we always do it the same way